Code/GraphMol/MolDraw2D/MolDraw2D.cpp

```diff
@@ -3847,7 +3847,7 @@ string MolDraw2D::getAtomSymbol(const RDKit::Atom &atom,
     symbol = getAtomListText(atom);
   } else if (isComplexQuery(&atom)) {
     symbol = "?";
-  } else if (drawOptions().atomLabelDeuteriumTritium &&
+  } else if (drawOptions().atomLabelDeuteriumTritium && drawOptions().isotopeLabels &&
```

Comment:
I'm not sure that `D` or `T` count as isotope labels. I think the whole idea of giving them separate symbols is that they are somehow special compared to a standard H.

Reply:
Yeah, I was unsure about that too. Note that both new flags default to `true` so unless you explicitly switch isotope labels off you will still see the `D` and `T` symbols. But I think it makes sense that if you specify `I want to see D and T` labels, that overrides any other isotope-related flags, so I'll remove that `if` clause.